File: Paradigmas Ling. Prog/Ativ-Cap5&6/Ativ4.java
import java.util.ArrayList;

public class Ativ4 {
    public static void main(String[] args) {
        ArrayList<Livro> livros = new ArrayList<>();

        livros.add(new Livro("The Way Of Kings", "Brandon Sanderson", 2010));
        livros.add(new Livro("Words Of Radiance", "Brandon Sanderson", 2018));
        livros.add(new Livro("Wind And Truth", "Brandon Sanderson", 2024));

        System.out.println("Títulos dos livros:");
        for (Livro l : livros) {
            System.out.println(l.titulo);
        }
    }
}

File: Paradigmas Ling. Prog/Ativ-Cap5&6/Ativ4.c
#include <stdio.h>
#include <string.h>

struct Livro {
    char titulo[50];
    char autor[50];
    int anoPublicacao;
};

int main() {
    int numeros[5] = {10, 20, 30, 40, 50};

    printf("Array de inteiros:\n");
    for (int i = 0; i < 5; i++) {
        printf("%d ", numeros[i]);
    }
    printf("\n\n");

    struct Livro livro1;
    strcpy(livro1.titulo, "The Way of Kings");
    strcpy(livro1.autor, "Brandon Sanderson");
    livro1.anoPublicacao = 2010;

    printf("Livro:\n");
    printf("Titulo: %s\n", livro1.titulo);
    printf("Autor: %s\n", livro1.autor);
    printf("Ano: %d\n", livro1.anoPublicacao);

    return 0;
}
